Added the design thinking section (#63)
zacaz90/Tutorial10_Group-1: Report.md

========================
*3. Describe your approach from a Design Thinking principles perspective. Your Design Thinking approach to address the problem with artefacts*

## **Approach - Design Thinking**
Design thinking is a non-linear, iterative process which seeks the understanding of users to challenge assumptions, redefine problems and create innovative solutions to prototype and test. This customer-centric approach was undertaken to address the improvements requested by the Major Travel Company for their Call Management Centre (CMC). The design thinking approach undertook consists of the execution of five phases, (1) empathize, (2) define, (3) ideate, (4) prototype and (5) test and by addressing these phrases we derived customer-centric innovation. 

### Empathise

> Phrase one consisted of empathising with stakeholders, this represents anyone who is impacted by the project being developed with a particular focus extended on the end customer. Empathy is a crucial step of achieving a human-centred design process and was utilised to gain insight into users and their needs without introducing any pre imposed bias. When we create solutions, we get better solutions when we think of those who they will affect. To document these perspectives the tool of an empathy map was utilised to collaboratively visualise and share our understanding of stakeholders needs and aid future decision making. This stage allowed us to gain a deeper understanding of the perspective of the users involved and how to shape our solution. 

Reviewer: Should start with "Phase" along with all the other phases ahaha
Also personally think "the tool of an empathy map" would flow better if it was just "To document these perspectives, empathy maps were utilised..."

Author: Right I didn't catch that, definitely, it should be phase - not "phrase". Will make that change. And yep the flow of "the tool of an empathy map" would flow better if it was just "To document these perspectives, empathy maps were utilised.." sounds better - will adjust to that

### Define

> Phrase two of defining allowed us to synthesis the knowledge obtained in the empathise stage and define the core problems to solve. Here the problem statements were directly derived from creating point of view (POV) statements for all the stakeholders identified. This staged allowed us as a team to consolidate assumptions and agree on features to be developed to solve the problem given. 

Reviewer: "Phase" while I'm here. Should be "synthesise"

Author: Ahhh my grammer keeps getting better - will be noted and change it to "synthesise"

### Ideate

> Phrase three consisted of utilising the user research conducted to ideate solutions to the problem statements we created in the prior stage. The brainstorming and worst possible idea techniques were utilised to assist within our ideation stage. These techniques were chosen to initiate freeform group discussions and simulated creative solutions in light of the problems identified. These sessions consisted of developing “how might we? (HMW)” statements and deriving features relative to the point of view statements developed in the define stage. The conclusions of the brainstormed were documented in (need to do this???). At the conclusion of this stage, the agreed features were converted to issues to be developed for the prototype and testing stage. 

Reviewer: Yeah, I'm pretty sure we'll need to document them in. I was going to do it but confused myself so I will ask what he wants from the 200-300 word reflection before attempting it.

Author: Yea I was also confused when I saw that session - since we did do brainstorming but the reflection is something we can to in today's lesson :)

### Prototype/Test

> The prototype and subsequent testing phrase is representative of the final stages within the design thinking approach. This consists of producing a small-scaled, mock-up of the product or specific features found within the product so that tangible investigations can occur on the problem solutions generated in the previous stage. To assist in the prototype stage the team designed artefacts representative of the design of the system and the user flow these are demonstrated within the use case diagram, activity diagrams, class diagram and collaborative diagrams provided. Due to the nature of this project, the prototype and respective testing are not within scope. 

Reviewer: "testing phase" ahahaha.
Hm also personally think just "investigations can occur on the solutions generated" rather than 'problem solutions' so it's just a little slimmer.
"design of the system and the user flow. These are demonstrated" kind of a run-on sentence, maybe split it into two?

### Artefacts
Artefact | Description
---|---

> Empathy Maps | Is a tool used to visualise user attitudes and behaviours, this map is split into 6 quadrants (Says, Thinks, Does, Feels, Pains and Gains).

Reviewer: ```suggestion
Empathy Maps | Is a tool used to visualise user attitudes and behaviours. This map is split into 6 quadrants (Says, Thinks, Does, Feels, Pains and Gains).
```

> POV Statements | Point of view statements are statements which focus on the specific user and their needs. These statements represent the teams results for the define phrase.

Reviewer: ```suggestion
POV Statements | Point of view statements are statements which focus on the specific user and their needs. These statements represent the team's results for the define phase.
```

> How might we? (H | How might we statements represent our results of the ideate phase and are short statements built from point of view statements

Reviewer: I'm just assuming here
```suggestion
How might we? (HMW) | How might we statements represent our results of the ideate phase and are short statements built from point of view statements
```
